pkg/controller/networkpolicy/crd_utils.go

```diff
@@ -61,6 +61,18 @@ func toAntreaServicesForCRD(npPorts []v1alpha1.NetworkPolicyPort, npProtocols []
 				ICMPCode: npProtocol.ICMP.ICMPCode,
 			})
 		}
+		if npProtocol.IGMP != nil {
+			curProtocol := controlplane.ProtocolIGMP
+
+			klog.V(4).InfoS("Get groupAddress", npProtocol.IGMP.GroupAddress,
```

Comment:
The format is wrong. key and value numbers are not equal, and I doubt the necessity of this log, no such log is ever needed when processing other protocols.

Reply:
removed